Makefile

```diff
@@ -128,8 +128,12 @@ ifeq ("$(TRAVIS_COVERAGE)", "1")
 	@echo "Running in TRAVIS_COVERAGE mode."
 	@export log_level=error; \
 	$(GO) get github.com/go-playground/overalls
-	$(OVERALLS) -project=github.com/pingcap/tidb -covermode=count -ignore='.git,vendor,cmd,docs,LICENSES,ddl/failtest,ddl/testutil/,executor/esqtest' \
-			-concurrency=1 || { $(GOFAIL_DISABLE); exit 1; }
+	$(OVERALLS) -project=github.com/pingcap/tidb \
+			-covermode=count \
+			-ignore='.git,vendor,cmd,docs,LICENSES' \
+			-concurrency=2 \
```

Comment:
Why update this flag? Not stated in the description of the PR.

Reply:
About `ignore`, because I am not sure `ddl/failtest,ddl/testutil/,executor/esqtest` can be ignored.( Discussed in slack).
About `concurrency`, this can make the test run faster a little.

Reply:
Sorry for my fault.
I have changed the description.